components/board/customlinux/setup_test.go

```diff
@@ -0,0 +1,27 @@
+//go:build linux
+
+// Package customlinux implements a board running linux
+package customlinux
+
+import (
+	"testing"
+
+	"go.viam.com/test"
+)
+
+func TestConfigParse(t *testing.T) {
+	emptyConfigPath := "./data/test_config_empty.json"
+	_, err := parseBoardConfig(emptyConfigPath)
```

Comment:
I'm also not keen on the name `parseBoardConfig`, since board configs get parsed elsewhere (in rdk/resource/config.go, I think?). Is this `parsePinDefinitions` or similar? Maybe not; it'll be more obvious when I can see the function.